### ISSUE TITLE
Breadcrumb doesn't work when url ends with a trailing slash

### ISSUE THREAD

**Issue by Extarys** [harlan-zw/nuxt-seo-kit#91]
### Describe the bug

When using `defineNuxtLink` with `trailingSlash: 'append'`, the breadcrumb stops showing page titles.

![image](https://github.com/harlan-zw/nuxt-seo-kit/assets/11581433/4abf58b7-3972-4afb-8746-03e55647333d)

To reproduce:

Create a NLink.ts component:
```
export default defineNuxtLink({
  componentName: 'NLink',
  trailingSlash: 'append'
})
```

or simply add a trailing slash to the current url, exemple: `127.0.0.1:3000/my/subdir/page1/`

### System / Nuxt Info

```Shell
System:
    OS: Linux 6.4 Fedora Linux 38 (Workstation Edition)
    CPU: (24) x64 AMD Ryzen 9 5900X 12-Core Processor
    Memory: 3.74 GB / 15.50 GB
    Container: Yes
    Shell: 5.2.15 - /bin/bash
  Binaries:
    Node: 20.5.1 - ~/.nvm/versions/node/v20.5.1/bin/node
    Yarn: 3.4.1 - ~/.nvm/versions/node/v20.5.1/bin/yarn
    npm: 9.8.0 - ~/.nvm/versions/node/v20.5.1/bin/npm
```

**Comment by Extarys** [comment]
Thanks for the quick fix! But, hmm, it's partially working. `breadcrumbTitle` doesn't seem to do anything when there is a trailing slash. The last breadcrumb is generated from the page's file name (default behaviour when `breadcrumbTitle` isn't set).
```
definePageMeta({
	breadcrumbTitle: 'My super title',
})
```

**Comment by harlan-zw** [comment]
Thanks! Missed this, should be fixed in 1.3.13
